hoot-rnd/src/main/cpp/hoot/rnd/cmd/CropRandomCmd.cpp

```diff
@@ -91,7 +91,7 @@ class CropRandomCmd : public BaseCommand
     const QString output = args[1].trimmed();
 
     bool ok = false;
-    const int maxNodes = args[2].toLong(&ok);
+    long maxNodes = args[2].toLong(&ok);
```

Comment:
This is passed into a function below as an integer, it should just be read from the command line as an integer.
```
const int maxNodes = args[2].toInt(&ok);
```